src/Polkadot/Extrinsic.cpp

```diff
@@ -91,6 +91,19 @@ Data Extrinsic::encodeCall(const Proto::SigningInput& input) {
             append(data, encodeCompact(value));
             // reward destination
             append(data, reward);
+        } else if (staking.has_bond_and_nominate()) {
+            // TODO
+            auto address = SS58Address(staking.bond_and_nominate().controller(), byte(input.network()));
+            auto value = load(staking.bond_and_nominate().value());
+            auto reward = byte(staking.bond_and_nominate().reward_destination());
+            // call index
+            append(data, getCallIndex(network, stakingBond));
```

Comment:
you can find the call index here: https://polkascan.io/polkadot/runtime-call/27-utility-batch